xbmc/interfaces/legacy/ModuleXbmc.h

```diff
@@ -404,6 +404,14 @@ namespace XBMCAddon
     void audioResume();
 
     /**
+     * getUserAgent() -- Returns Kodis HTTP UserAgent string
```

Comment:
Almost. That rule only applies to "it" - it's raining, its umbrella.

For everything else, possession always has an apostrophe.

Ironically, many native speakers get the "it" exception wrong and write it's instead of its :)